test/utils/ApiClient.spec.js

```diff
@@ -489,5 +492,46 @@ describe("utils/ApiClient", () => {
             });
             callback(new Error(), rawRes);
         });
+
+        it("should call callback with 'body' and 'res' and application/jose+json Content-Type", (cb) => {
+            const client = new ApiClient("test-username", "test-password", "test-server");
+
+            const rawRes = {
+                body: "test",
+                status: 200,
+                type: "application/jose+json",
+            };
+
+            const callback = client.wrapCallback((err, body, res) => {
+                expect(err).to.be.undefined();
+
+                body.should.be.equal("test");
+                rawRes.should.be.deep.equal(res);
+
+                cb();
+            });
+            callback(undefined, rawRes);
+        });
+
+        it("should call callback with static error message as 'errors', when Content-Type is wrong", (cb) => {
+            const client = new ApiClient("test-username", "test-password", "test-server");
+
+            const rawRes = {
+                body: "test",
+                status: 200,
+                type: "wrongContentType",
+            };
+
+            const callback = client.wrapCallback((err, body, res) => {
+                err.should.be.deep.equal([{
+                    message: "Invalid Content-Type specified in Response Header",
+                }]);
+                body.should.be.equal("test");
+                rawRes.should.be.deep.equal(res);
+
+                cb();
+            });
+            callback(new Error(), rawRes);
+        });
```

Comment:
I rechecked the code based on requirements from the ticket, it seems like NodeJS SDK does not support encrypted request for payloads like `application/jose+json`
for example in ApiClient doGet:113 

it has this request
`request.accept("json")` -> `request.set('Accept', 'application/json')` 
if not encrypted request and `request.set('Accept', 'application/jose+json'` if encrypted respectively

Reply:
Until PR #7 is not merged we can't define if request is encrypted or not so for now Accept header is always = 'application/json'